pom.xml

```diff
@@ -71,6 +71,8 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-support</artifactId>
+<!--            TODO until in bom-->
```

Comment:
https://github.com/jenkinsci/bom/pull/1422

Reply:
latest bom applied in [b21a47e](https://github.com/jenkinsci/pipeline-stage-step-plugin/pull/70/commits/b21a47e2d714b6ee0d4ff7590401fd27af583e65)